packages/calcite-components/src/components/alert/alert.tsx

```diff
@@ -353,6 +359,20 @@ export class Alert implements OpenCloseComponent, LoadableComponent, T9nComponen
     this.determineActiveAlert();
   }
 
+  // when an alert is unregistered, trigger a queue sync
```

Comment:
I think we can drop this comment since the code is pretty expressive. Maybe it'd be better to add a note on why we need to dispatch on the window instead.